build.sbt

```diff
@@ -207,10 +207,7 @@ lazy val tests = project
     skip in publish := true,
     libraryDependencies ++= Seq(
       // Test dependencies
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, 13)) => "com.lihaoyi" %% "scalatags" % "0.9.3"
```

Comment:
do you know why we had a split? what does the commit message say, if you use git blame?

Reply:
Looks like it was introduced while adding support for 2.13 https://github.com/scalameta/scalafmt/pull/1522

Doesn't seem there is any particular reason other than maybe it was only released for 2.13?

Reply:
last question. should we keep this check for 2.13 and just update its version? just in case the next version of scalatags will once again not support an older version of scala, and we won't need to remember how to write this construct?

Reply:
I don't think they should soon start to stop supporting it, but we have this construct all over the place in Metals and I think this is also done for parallel-collections https://github.com/scalameta/scalafmt/blob/master/build.sbt#L119